Package.swift

```diff
@@ -599,8 +599,12 @@ let package = Package(
       exclude: [
         "DefaultUI/CHANGELOG.md",
         "DefaultUI/README.md",
+        "Resources/tvOS"
```

Comment:
Seems like we still have to exclude the tvOS resources (else build failure when building on iOS targets) despite having the condition for this target set for iOS platforms only. Since FIAM only supports iOS, this should be okay?

Reply:
Should be fine until we're ready to restart the tvOS work

Reply:
@ncooke3 @paulb777 FIAM currently supports tvOS-- there's no default UI, but devs can define custom interfaces.

Reply:
@christibbs Oh right. Sorry about my confusion.

Reply:
FIAM support is there for CocoaPods but not Swift Package Manager : https://github.com/firebase/firebase-ios-sdk/blob/master/Package.swift#L585

Reply:
I referenced https://firebase.google.com/docs/ios/learn-more ...should this table be amended to include a check for FIAM / tvOS?

Reply:
@ncooke3 Yes, reach out to Mark Arndt to update this.